deepspeed/runtime/lr_schedules.py

```diff
@@ -381,6 +381,12 @@ def get_lr(self):
             lr_range_test_min_lr * lr_increase for lr_range_test_min_lr in self.min_lr
         ]
 
+    def get_last_lr(self):
+        """ Return last computed learning rate by current scheduler.
+        """
+        assert getattr(self, '_last_lr', None), "need to call step() first"
```

Comment:
Actually, I think this is still weak, since if last lr is 0 it will fail. Probably should change it:
```
assert getattr(self, '_last_lr', False), "need to call step() first"
```

Reply:
I changed it to the latter.

Reply:
Agreed. Good catch.